modules/hal_nordic/nrfx/nrfx_config_reserved_resources.h

```diff
@@ -0,0 +1,710 @@
+/*
+ * Copyright (c) 2024, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NRFX_CONFIG_RESERVED_RESOURCES_H__
+#define NRFX_CONFIG_RESERVED_RESOURCES_H__
+
+/** @brief Bitmask that defines GPIOTE130 channels reserved for use outside
+ *         of the nrfx library.
+ */
+#define NRFX_GPIOTE130_CHANNELS_USED                                                               \
+	(~NRFX_CONFIG_MASK_DT(DT_NODELABEL(gpiote130), owned_channels) |                           \
+	 NRFX_CONFIG_MASK_DT(DT_NODELABEL(gpiote130), child_owned_channels))
+
+/** @brief Bitmask that defines GPIOTE131 channels reserved for use outside
+ *         of the nrfx library.
+ */
+#define NRFX_GPIOTE131_CHANNELS_USED                                                               \
+	(~NRFX_CONFIG_MASK_DT(DT_NODELABEL(gpiote131), owned_channels) |                           \
+	 NRFX_CONFIG_MASK_DT(DT_NODELABEL(gpiote131), child_owned_channels))
+
+/** @brief Bitmask that defines EGU instances that are reserved for use outside
+ *         of the nrfx library.
+ */
+#define NRFX_EGUS_USED 0
+
+/** @brief Bitmask that defines TIMER instances that are reserved for use outside
+ *         of the nrfx library.
+ */
+#define NRFX_TIMERS_USED 0
+
+/*
+ * The enabled Bluetooth controller subsystem is responsible for providing
+ * definitions of the BT_CTLR_USED_* symbols used below in a file named
+ * bt_ctlr_used_resources.h and for adding its location to global include
+ * paths so that the file can be included here for all Zephyr libraries that
+ * are to be built.
+ */
+#if defined(CONFIG_BT_LL_SW_SPLIT)
+#include <bt_ctlr_used_resources.h>
+#if defined(DPPI_PRESENT)
+#if defined(NRF53_SERIES)
+#define NRFX_DPPI0_CHANNELS_USED_BY_BT_CTLR BT_CTLR_USED_PPI_CHANNELS
+#define NRFX_DPPI0_GROUPS_USED_BY_BT_CTLR   BT_CTLR_USED_PPI_GROUPS
+#elif defined(LUMOS_XXAA)
+#define NRFX_DPPI10_CHANNELS_USED_BY_BT_CTLR BT_CTLR_USED_PPI_CHANNELS
+#define NRFX_DPPI10_GROUPS_USED_BY_BT_CTLR   BT_CTLR_USED_PPI_GROUPS
+#endif
+#else /* defined(DPPI_PRESENT) */
+#define NRFX_PPI_CHANNELS_USED_BY_BT_CTLR BT_CTLR_USED_PPI_CHANNELS
+#define NRFX_PPI_GROUPS_USED_BY_BT_CTLR   BT_CTLR_USED_PPI_GROUPS
+#endif /* defined(DPPI_PRESENT) */
+#endif /* defined(CONFIG_BT_LL_SW_SPLIT) */
+
```

Comment:
What about including
https://github.com/nrfconnect/sdk-nrfxlib/blob/main/mpsl/include/mpsl.h
and providing
`NRFX_xxxxxx_CHANNELS_USED_BY_MPSL` based on `MPSL_xxxx_CHANNELS_USED_MASK`
https://github.com/nrfconnect/sdk-nrfxlib/blob/main/mpsl/include/mpsl.h#L51-L54

Reply:
This won't be added in upstream.